### PR TITLE
Fix path to LICENSE in README.md

### PR DESCRIPTION
**Pull request by maxhbr**
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/940)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,4 +180,4 @@ The most important root project Gradle tasks are listed in the table below.
 
 Copyright (C) 2017-2018 HERE Europe B.V.
 
-See the [LICENSE](../LICENSE) file in the root of this project for license details.
+See the [LICENSE](LICENSE) file in the root of this project for license details.
```
Comment (sschuberth) [inline README.md]: Thanks for this fix! Mind writing the path as `./LICENSE` for clarity and consistency with similar code in this file?

**Comment by maxhbr** [comment]
I will not be able to sign the CLA without huge overhead.

But since this PR obviously does not contain any intellectual property, I am sure someone will find a way to apply the fix.

This PR can be closed.

**Comment by sschuberth** [comment]
> I will not be able to sign the CLA without huge overhead.

@tsteenbe, yet another reason to switch to a DCO instead.

> This PR can be closed.

I've incorporated your fix into PR https://github.com/heremaps/oss-review-toolkit/pull/942.

**Comment by sschuberth** [comment]
> I will not be able to sign the CLA without huge overhead.

@maxhbr, I'm happy to announce that we're [using the DCO](https://probot.github.io/apps/dco/) now instead, which should easy corporate approval. Looking forward to your future contributions! 😉

**Pull request closed without merging**